addons/web/static/src/js/chrome/keyboard_navigation_mixin.js

```diff
@@ -30,7 +30,10 @@ odoo.define('web.KeyboardNavigationMixin', function (require) {
             'keyup': '_onKeyUp',
         },
 
-        init: function () {
+        init: function (options) {
```

Comment:
Could be worth adding the docstring since you introduce a parameter.

\+ this file is in web/ so mention it in modified apps in the commit message